style.css

```diff
@@ -458,3 +471,568 @@ footer div.icons form > button:focus {
   background-color: #e05304;
   color: #f4f1f0;
 }
+
+@media (min-width: 320px) {
+  .about-container {
```

Comment:
Your `about` section is spanning more than 1 vertical screen. Kindly fix it as per the Figma design. Please use the same background colors, padding, margin, font size, etc as in the Figma design. Also, the horizontal line is missing in your design. Kindly include it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/186838468-551853bc-f9e3-422f-bb22-5b457a67a2b7.png) | ![image](https://user-images.githubusercontent.com/33268927/186838312-3794bd70-066d-400c-9ff1-ee461ca57e18.png) |

Reply:
- [ ] Kindly make sure to implement the suggested changes by adding the correct background color to the `about` section.